.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "proto/opentelemetry-proto"]
+	path = proto/opentelemetry-proto
+	url = git@github.com:open-telemetry/opentelemetry-proto.git
```

Comment:
Should we restrict to a specific tag?

For example: `branch = tags/v1.5.0`

Reply:
I checked out that tag in the submodule and committed. I believe that does what you're asking, though this file will remain as-is.

Reply:
Fixed.

Reply:
@jmacd I was asking to pin it to a specific tag, for a consistent experience when calling `git submodule update`. That's the reason to explicitly add `branch = tags/v.1.5.0`. We follow this in the Rust SDK repo: https://github.com/open-telemetry/opentelemetry-rust/blob/main/.gitmodules